libr/parse/p/parse_riscv_pseudo.c

```diff
@@ -181,10 +181,11 @@ static int parse(RParse *p, const char *data, char *str) {
 					ptr = (char*)r_str_lchr (buf, ']');
 					if (n && ptr) {
 						char *rest = strdup (ptr + 1);
+						int dist = ptr - buf;
```

Comment:
Why not hold the result in a variable instrad of computing it twice below?

Also. The type must be size_t

Reply:
Alright, so, I'll make it:
`size_t dist = len + 1 - ptr - buf;`